pythonforandroid/toolchain.py

```diff
@@ -301,6 +309,13 @@ def __init__(self):
                   'Python modules'),
             default='')
 
+        generic_parser.add_argument(
+            '--blacklist',
+            help=('Blacklist an internal recipe from use. Allows '
+                  'disabling Python 3 core modules to save size'),
+            dest="blacklist",
+            default='')
```

Comment:
why isn't the default `None` or `[]`?

Reply:
@AndreMiras it's a string (comma separated list)